packages/react-admin/src/sideEffect/saga/crudResponse.js

```diff
@@ -72,6 +89,17 @@ function* handleResponse({ type, requestPayload, error, payload }) {
                       ),
                   ])
                 : yield [put(showNotification('ra.notification.deleted'))];
+        case CRUD_DELETE_MANY_SUCCESS: {
+            const actions = [put(showNotification('ra.notification.deleted'))];
```

Comment:
Same as above

Reply:
I agree, although this is an advanced feature that we can delay for now.

Reply:
The response should return the ids of the updated/deleted items. So it should be easy to determine which have failed in a custom saga